syntax = "proto3";

package tvsc.service.hello;

service Hello {
  rpc say_hello(HelloRequest) returns (HelloReply);
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string msg = 1;
}
